packages/cdk/lib/wes_adapter/install.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+rm -rf ./package && mkdir ./package
+pip3 install -r requirements.txt --target ./package && cd ./package && zip -r ../wes_adapter.zip .
+cd ..
```

Comment:
I'm not a huge fan of jumping around directories like this. It's hard to track and can lead to weird bugs if a developer forgets what the current working dir is.

Consider wrapping the call in parentheses to avoid changing the working dir: `(cd ./package && zip -r ../wes_adapter.zip .)`

Reply:
Done, Thanks